Library/Formula/mesos.rb

```diff
@@ -12,19 +12,22 @@ class Mesos < Formula
   depends_on :java => "1.7+"
   depends_on :macos => :mountain_lion
   depends_on "maven" => :build
-  # Use our Zookeeper for Yosemite and not the one shipped with Mesos
-  # Remove with next release.
-  # See https://github.com/Homebrew/homebrew/issues/32965
-  depends_on "zookeeper" if MacOS.version == :yosemite
+  depends_on :apr => :build
+  depends_on "subversion"
+  
+
+  needs :cxx11
 
   def install
     args = ["--prefix=#{prefix}",
             "--disable-debug",
             "--disable-dependency-tracking",
             "--disable-silent-rules",
+            "--with-svn=/usr/local/Cellar/subversion/1.8.11",
```

Comment:
Yup :+1:. Homebrew will interpret that as ` HOMEBREW_PREFIX/opt/subversion ` which makes it prefix-neutral. The mechanism doesn't care where people put Homebrew, It will always point to the right location regardless. ` opt/subversion ` is actually a symlink to ` HOMEBREW_CELLAR/subversion/version/ `. So technically, ` #{Formula["subversion"].opt_prefix} ` = the Cellar directory.